rqt_shell/src/rqt_shell/shell.py

```diff
@@ -85,9 +86,16 @@ def __init__(self, context):
         super(Shell, self).__init__(context)
         self._context = context
         self.setObjectName('Shell')
+        self.args = self._parse_args(context.argv())
 
         self._widget = None
 
+    def _parse_args(self, argv):
+        parser = argparse.ArgumentParser(prog='rqt_shell', add_help=False)
```

Comment:
Hm, it might have a reason why it is there but I don't recall it atm. Then its probably better to keep it. I checked that the new option still shows up with `--help` even when the argument is set.